crates/sui-config/src/node.rs

```diff
@@ -560,9 +560,9 @@ pub struct AuthorityStorePruningConfig {
     /// number of epochs to keep the latest version of transactions and effects for
     #[serde(skip_serializing_if = "Option::is_none")]
     pub num_epochs_to_retain_for_checkpoints: Option<u64>,
-    /// enables pruner to prune no longer needed object tombstones. We don't serialize it if it is the default value, false.
+    /// disables object tombstone pruning. We don't serialize it if it is the default value, false.
     #[serde(default, skip_serializing_if = "std::ops::Not::not")]
-    pub enable_pruning_tombstones: bool,
+    pub killswitch_tombstone_pruning: bool,
```

Comment:
Why not just call it disable_pruning_tombstones?

Reply:
Yeah, that sounds good too. I simply copy pasted @phoenix-o 's suggestion. I don't have any preference, so I see if anyone else has opinions about the naming.